beamer/config/commands.py

```diff
@@ -58,21 +58,21 @@ def _replay_event(w3: Web3, deployment: Deployment, config: Configuration, event
                 symbol = token.functions.symbol().call()
             except ContractLogicError:
                 # If token symbol is not available, use address instead
-                symbol = event.event_address
+                symbol = event.token_address
+
+            # If the symbol already exists for a different token address,
+            # default to use token address as a symbol
+            if config.token_addresses.get(symbol, event.token_address) != event.token_address:
+                symbol = event.token_address
 
             if removal:
-                config.request_manager.tokens.pop(symbol)
-                config.token_addresses.pop(symbol)
+                config.request_manager.tokens.pop(symbol, None)
+                config.token_addresses.pop(symbol, None)
```

Comment:
Apart from the first call being
```
updateToken(address, 0, 0)
```
is there any other case where these entries may not exist?

Reply:
Sure, since the above code piece is the production code, I could simply do this call on chain and the code would break.